apps/pipelines/views.py

```diff
@@ -83,13 +84,22 @@ def get_context_data(self, **kwargs):
         llm_providers = LlmProvider.objects.filter(team=self.request.team).values("id", "name", "type").all()
         llm_provider_models = LlmProviderModel.objects.for_team(self.request.team).all()
         pipeline = Pipeline.objects.get(id=kwargs["pk"], team=self.request.team)
+        exclude_services = [SyntheticVoice.OpenAIVoiceEngine]
+        if flag_is_active(self.request, "flag_open_ai_voice_engine"):
+            exclude_services = []
+        synthetic_voices = SyntheticVoice.get_for_team(self.request.team, exclude_services=exclude_services)
         return {
             **data,
             "pipeline_id": kwargs["pk"],
             "pipeline_name": pipeline.name,
             "node_schemas": _pipeline_node_schemas(),
             "parameter_values": _pipeline_node_parameter_values(
-                self.request.team, llm_providers, llm_provider_models, include_versions=pipeline.is_a_version
+                team=self.request.team,
+                llm_providers=llm_providers,
+                llm_provider_models=llm_provider_models,
+                synthetic_voices=synthetic_voices,
```

Comment:
You could pass in an empty list here instead since the filtering in the function won't match anything. Then you can also remove the query above.
```suggestion
                synthetic_voices=[],
```